### PR TITLE
Feature/network indication 222

### LINKED ISSUE #222: wrong network indication
when user is using metamask on any network other than "mainnet", please include this warning message.
(relevant mostly for data as the page is blank when metamask is connected to ropsten)

https://app.zeplin.io/project/5d70f6905eefcd18e1c40502/screen/5f7d6f5e4d2dc806216e721e/

### PR DESCRIPTION
**Pull request by sirius651**
Fixed #222 , #723
스크린샷 2020-12-16 오후 3 00 58

### CODE REVIEW

**Review by pingustar — CHANGES_REQUESTED** [review]
looking good, two small things I would change:

image

- the button styling looks off
- lets try to center the alert

I just thought about another possible solution. Just an idea, but what if we use the blue top bar "This app is in beta..." and replace it with a red bar "Unsupported Network ..."

File: src/store/modules/swap/ethBancor.ts [hunk 1]
```diff
@@ -2352,6 +2352,10 @@ export class EthBancorModule
     this.setTolerance(tolerance);
   }
 
+  get isCurrentMainnet() {
```
Comment (Velua) [inline src/store/modules/swap/ethBancor.ts:2355]: Perhaps ```isSupportedNetwork``` instead.

**Review by Velua — CHANGES_REQUESTED** [review]
Let's make sure we display a warning for Ropsten as that's supported. See [here](https://github.com/bancorprotocol/webapp/issues/222#issuecomment-704820762)

**Comment by sirius651** [comment]
> looking good, two small things I would change:
> 
> image
> 
> * the button styling looks off
> * lets try to center the alert
> 
> I just thought about another possible solution. Just an idea, but what if we use the blue top bar "This app is in beta..." and replace it with a red bar "Unsupported Network ..."

Good suggestion. that would be more simple. I just followed the screenshot : https://github.com/bancorprotocol/webapp/issues/222#issuecomment-745123419

File: src/store/modules/swap/ethBancor.ts (hunk 1 above)
Comment (sirius651) [reply src/store/modules/swap/ethBancor.ts]: Changed it. and warning appears only if it is not the mainnet.

**Comment by sirius651** [comment]
스크린샷 2020-12-17 오전 10 39 28
Fix weird css stuff

**Review by pingustar — CHANGES_REQUESTED** [review]
on mobile I am getting css issues.
image

and the ropsten warning that @Velua mentioned is missing.

I have quickly put something together using the top bar instead of an alert.

What do you guys think? @sirius651 @Velua 

Have a look at https://github.com/bancorprotocol/webapp/pull/739

**Comment by sirius651** [comment]
> on mobile I am getting css issues.
> image
> 
> and the ropsten warning that @Velua mentioned is missing.
> 
> I have quickly put something together using the top bar instead of an alert.
> 
> What do you guys think? @sirius651 @Velua
> 
> Have a look at #739

yes, because sentence is long, it looks like that on mobile.
I think your suggestion would be more straightforward. @pingustar

**Comment by pingustar** [comment]
going with the quick and easy solution of replacing the blue top bar with the warning/error. Ropsten will show a warning, and unsupported networks will show an error.

**Pull request merged**